app/javascript/controllers/application.js

```diff
@@ -0,0 +1,9 @@
+import { Application } from "@hotwired/stimulus"
+
+const application = Application.start()
+
```

Comment:
- [ ] As required for the project kindly make sure you have all the listed files and component in the project requirements, you should:
  1. Ensure that everything exists in the `app/javascript/application.js` file.
  2. Create the `App` component with react-router and render it in your static view.
  3. Create the `Greeting` component that will display a greeting. Set it up as a route in your App component.
  4. Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
  5. Display the random greeting in your Greeting component.

- [ ] Once you have fixed all the above changes, Kindly ensure that your two apps work together as this is the major requirement of the project.
You can create 5 random greeting in your database then:
   * run the rails API on your machine (**IMPORTANT: start the app with** `./bin/dev` command)
   * change the API url in the React app to your localhost
   * verify that React app uses the url and displayes a random greeting